lib/hal/led.h

```diff
@@ -0,0 +1,18 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+#include <hal/xbox.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+#define LED_OFF    0x00
+#define LED_GREEN  0x01
+#define LED_RED    0x10
+#define LED_AMBER  0x11
```

Comment:
Shouldn't these be binary?

```c
#define LED_OFF   0x0
#define LED_GREEN 0x1
#define LED_RED   0x2
#define LED_AMBER 0x3 //FIXME: Use ORANGE?
```

Reply:
I opted for HEX for easy OR'ing.

Reply:
Your old code was right, the LEDs are split into two different nibbles (one nibble for RED, one for GREEN).